pkg/logql/downstream_test.go

```diff
@@ -34,21 +34,22 @@ func TestMappingEquivalence(t *testing.T) {
 		query       string
 		approximate bool
 	}{
-		{`1`, false},
-		{`1 + 1`, false},
-		{`{a="1"}`, false},
-		{`{a="1"} |= "number: 10"`, false},
-		{`rate({a=~".+"}[1s])`, false},
-		{`sum by (a) (rate({a=~".+"}[1s]))`, false},
-		{`sum(rate({a=~".+"}[1s]))`, false},
-		{`max without (a) (rate({a=~".+"}[1s]))`, false},
-		{`count(rate({a=~".+"}[1s]))`, false},
-		{`avg(rate({a=~".+"}[1s]))`, true},
-		{`avg(rate({a=~".+"}[1s])) by (a)`, true},
-		{`1 + sum by (cluster) (rate({a=~".+"}[1s]))`, false},
-		{`sum(max(rate({a=~".+"}[1s])))`, false},
-		{`max(count(rate({a=~".+"}[1s])))`, false},
-		{`max(sum by (cluster) (rate({a=~".+"}[1s]))) / count(rate({a=~".+"}[1s]))`, false},
+		//{`1`, false},
+		//{`1 + 1`, false},
+		//{`{a="1"}`, false},
+		//{`{a="1"} |= "number: 10"`, false},
+		//{`rate({a=~".+"}[1s])`, false},
+		//{`sum by (a) (rate({a=~".+"}[1s]))`, false},
+		//{`sum(rate({a=~".+"}[1s]))`, false},
+		//{`max without (a) (rate({a=~".+"}[1s]))`, false},
+		//{`count(rate({a=~".+"}[1s]))`, false},
+		//{`avg(rate({a=~".+"}[1s]))`, true},
+		//{`avg(rate({a=~".+"}[1s])) by (a)`, true},
+		//{`1 + sum by (cluster) (rate({a=~".+"}[1s]))`, false},
+		//{`sum(max(rate({a=~".+"}[1s])))`, false},
+		//{`max(count(rate({a=~".+"}[1s])))`, false},
+		//{`max(sum by (cluster) (rate({a=~".+"}[1s]))) / count(rate({a=~".+"}[1s]))`, false},
```

Comment:
Why are these commented out?

Reply:
ugh, had some stuff on my working directory I didn't notice when I made this... fixed..